src/jit/compiler.cpp

```diff
@@ -4430,6 +4432,7 @@ bool Compiler::compRsvdRegCheck(FrameLayoutState curState)
 #if defined(_TARGET_ARM64_)
 
     // TODO-ARM64-CQ: update this!
+    JITDUMP(" Returning true (ARM64)\n\n");
     return true; // just always assume we'll need it, for now
```

Comment:
You might want to add a note that on Arm64 the definition is, so REG_IP1 (or x17)
`  #define REG_OPT_RSVD             REG_IP1`

Reply:
That is currently dumped by the caller; I could have added it to each return, but that would require duplicating it everywhere.